src/Client/UpgradedResponse.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace React\Http\Client;
+
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use React\Stream\DuplexStreamInterface;
+
+class UpgradedResponse
```

Comment:
This adds a new class as part of our public API in the internal `React\Http\Client` namespace.